php/src/Google/Protobuf/Internal/RepeatedField.php

```diff
@@ -138,6 +140,7 @@ public function offsetGet($offset)
      * @throws \ErrorException Non-existing index.
      * @throws \ErrorException Incorrect type of the element.
      */
+    #[\ReturnTypeWillChange]
     public function offsetSet($offset, $value)
```

Comment:
```suggestion
    public function offsetSet($offset, $value): void
```
According to doc, any return value will be ignored, so it seems to be safe to use "void" return type hint even without `#[\ReturnTypeWillChange]` attribute as like `Iterator::next`

Reply:
@Tony-Sol - I did have have void return types originally, but removed them to retain BC to php 7.0 (some discussion above on the subject). The void return type only [arrived in 7.1](https://wiki.php.net/rfc/void_return_type), and the tests were failing (from memory, searching for and not finding a void class in the current namespace)